semantic-conventions/syntax.md

```diff
@@ -7,17 +7,20 @@ Then, the semantic of each field is described.
 
 <!-- toc -->
 
-- [JSON Schema](#json-schema)
```

Comment:
I think there is no need to regenerate the TOC.

Reply:
vscode auto-regenerates it for me. Anyway, returned original one back